tool/src/org/antlr/v4/codegen/model/OutputFile.java

```diff
@@ -26,7 +26,7 @@ public OutputFile(OutputModelFactory factory, String fileName) {
         super(factory);
         this.fileName = fileName;
         Grammar g = factory.getGrammar();
-		grammarFileName = g.fileName;
+		grammarFileName = g.fileName.replace("\\", "/"); // Prevent a path with windows delim and u breaking Java pre-parser on comments
```

Comment:
As I understand, this bug is only relevant to Java target. Is it possible to fix the path only for Java target on Windows platform?